### DISCUSSION TITLE
Error with Post Install User Method

### DISCUSSION THREAD

**Discussion by josmirpa** [telekom-security/tpotce#1234] [Q&A]
When I try to install T-Pot on a Debian completely clean, I get this error when I install it with "Post Install User Method" and "Post Install Auto Method":

![2022-12-20 15_27_14-Debian T-Pot  Corriendo  - Oracle VM VirtualBox](https://user-images.githubusercontent.com/71086685/208714697-3d0549ab-6c5b-40d7-8116-2d2b5ef8159c.png)

If I change in line 543 myPASS2="pass2" to "myPASS2="pass1" the error is fixed but I can't login to some services.

I don't know if this is a bug or it's my fault.

Thanks!

**Comment by t3chn0m4g3** [comment]
Please check the ReadMe and use the Debian Netinstall Image.

**Reply by josmirpa** [reply]
Is there no way to install the program other than with the iso image?

**Reply by t3chn0m4g3** [reply]
Sure, you can even run a post install on a full blown Debian 11 installation. Thing is (as mentioned in #1232) you then need to take care of all the services potentially occupying ports T-Pot needs to work correctly.
